7_drivers_munge/src/GCM_driver_nc_utils.R

```diff
@@ -0,0 +1,27 @@
+# Munging driver data from geoknife to create NetCDF for each GCM.
+
+#' @title Creates a NetCDF file with GCM driver data
+#' @description For each of the GCMs, create a single NetCDF file containing all the
+#' variables and all the grid cells.
+#' @param nc_file netcdf file name output
+#' @param gcm_raw_files vector of feather file paths, one for each tile with data for the current GCM.
+#' @param dim_time_input vector of GCM driver data dates
+#' @param dim_cell_input vector of all GCM driver grid cells (whether or not data was pulled)
+#' @param vars_info variables and descriptions to store in NetCDF
+#' @param global_att global attribute description for the observations (e.g. notaro_ACCESS_1980_1999)
```

Comment:
Looks like you may have tweaked the format of this global attribute description since providing this example.

Reply:
Oh yes. I should probably just delete all of this documentation since I anticipate arguments passed to the function will change immensely with #252 but this runs in the pipeline, so was keeping for now.